packages/api/src/env.js

```diff
@@ -26,19 +26,16 @@ import pkg from '../package.json'
 export function envAll (_, env, event) {
   env.sentry = (env.SENTRY_DSN || typeof SENTRY_DSN !== 'undefined') && new Toucan({
     dsn: env.SENTRY_DSN || SENTRY_DSN,
-    event,
+    context: event,
     allowedHeaders: ['user-agent'],
     allowedSearchParams: /(.*)/,
     debug: false,
     rewriteFrames: {
       root: '/'
     },
-    version: env.VERSION || VERSION,
-    pkg: {
-      ...pkg,
-      // sentry cannot deal with "/" in version
-      name: pkg.name.replace('@web3-storage/', '')
-    }
+    environment: env.ENV || ENV,
+    release: env.SENTRY_RELEASE || SENTRY_RELEASE,
+    pkg
```

Comment:
No need to replace the package.name now as sentry will use the value of the `release` option now instead of the package name.